tianshou/utils/net/discrete.py

```diff
@@ -163,3 +163,36 @@ def forward(
         x = x.view(-1, self.num_atoms).softmax(dim=-1)
         x = x.view(-1, np.prod(self.action_shape), self.num_atoms)
         return x, state
+
+
+class QRDQN(DQN):
+    """Reference: Distributional Reinforcement Learning with Quantile Regression.
+
+    For advanced usage (how to customize the network), please refer to
+    :ref:`build_the_network`.
+    """
+
+    def __init__(
+        self,
+        c: int,
```

Comment:
are you sure `c`, `h`, `w` are descriptive enough ?

Reply:
These three are the same as existing DQN in discrete.py. I just changed the output dimension of DQN.